test/hotspot/jtreg/compiler/calls/common/InvokeDynamicPatcher.java

```diff
@@ -73,11 +153,7 @@ public static void main(String args[]) {
         } catch (URISyntaxException ex) {
             throw new Error("TESTBUG: Can't get code source" + ex, ex);
         }
-        try (FileInputStream fis = new FileInputStream(filePath.toFile())) {
```

Comment:
Don't you have to delete lines 149-155 also?

Reply:
This path is used to dump the destination bytes. I tried to use `test.classes` to dump the output bytes but to no avail - seems libraries have some custom mechanisms, that I can neither find the location of their classes, nor can I easily pass javac flags for libraries via jtreg.